features/polyExplorer/src/styles.css

```diff
@@ -49,13 +49,6 @@
     src: url("../fonts/jost_bold.ttf");
 }
 
-* {
```

Comment:
Nice! Now we can get rid of a lot of `!important` that we are using and we don't need anymore